octodns/provider/ns1.py

```diff
@@ -224,11 +228,13 @@ class Ns1Provider(BaseProvider):
         'NA': ('US-CENTRAL', 'US-EAST', 'US-WEST'),
     }
 
-    def __init__(self, id, api_key, retry_count=4, *args, **kwargs):
+    def __init__(self, id, api_key, retry_count=4, manage_root_ns=False, *args,
+                 **kwargs):
```

Comment:
I believe there's no need to include `manage_root_ns` here as an explicit parameter and pass it through to the `super` call, it should be in `**kwargs`. This happens in a few other places as well.